### ISSUE TITLE
module does not show up if connection is not made

### ISSUE THREAD

**Issue by Tedcharlesbrown** [bitfocus/companion-module-barco-eventmaster#70]
Cannot pre-program with module unless connection has been made to Event Master

**Comment by JeffreyDavidsz** [comment]
That is correct, the presets are loaded from the system, so with no system, no presets. You could fire up the simulator and pre-program if needed.
